content/how-to/account-management/change-account-information/index.md

```diff
@@ -72,13 +83,13 @@ provide the following information:
 
 ### Best practices for legal change forms
 
-The Service Transfer and Primary Contact Change forms have many required
-fields that must be complete before the Billing team can process the forms.
+The **Service Transfer change forms** and **Primary Contact Change forms** have many required
+fields that must be completed before the Billing team can process the forms.
 After you have completed the form, scan the document and attach it to the
 support ticket in the [Cloud Control Panel](https://login.rackspace.com) or
 the [MyRackspace Control Panel](https://login.rackspace.com).
```

Comment:
We should probably avoid mentioning Cloud Control Panel, and MyRackspace Control panel for this. All tickets are now submitted through Fabric Ticketing. In the future we are suggesting for them to use Support Center which redirects to Ticketing but bypasses step 1 categories.
So perhaps we can mention go to support center, submit issue button, this will redirect to Ticketing to step 2, and be clear to mention for them to create a "Custom Ticket" and attach file in step 3. Add link to Fabric Ticketing for more information on creating a custom ticket. I hope we have a section for that.